### ISSUE TITLE
idempotent-json and univ-config modules do not exist.

### ISSUE THREAD

**Issue by matthova** [ORESoftware/hr4R#3]
Not sure what happened, but both of these modules were revoked by NPM for security reasons
https://www.npmjs.com/package/univ-config
https://www.npmjs.com/package/idempotent-json

**Comment by ORESoftware** [comment]
Ah thanks, yeah I had to remove and then rename them because they were originally IP of the company I worked for. I need to fix this.

**Comment by marekkaczkowski** [comment]
@matthova idempotent-json --> https://github.com/the1mills/siamese

**Comment by ORESoftware** [comment]
I wouldn't use this project as a reference, the article though still stands strong, but this project is outdated. The hot-reloading mechanism though in the project is the only important part.

**Comment by marekkaczkowski** [comment]
For sure, but I'd like to experiment a bit with old stack :)

**Comment by ORESoftware** [comment]
yeah, I am digging Angular5, but it doesn't have the dynamic loading properties of RequireJS or SystemJS, and the build is so complex compared to the old way lol
